crates/teloxide/examples/buttons.rs

```diff
@@ -107,17 +108,17 @@ async fn inline_query_handler(
 /// **IMPORTANT**: do not send privacy-sensitive data this way!!!
 /// Anyone can read data stored in the callback button.
 async fn callback_handler(bot: Bot, q: CallbackQuery) -> Result<(), Box<dyn Error + Send + Sync>> {
-    if let Some(version) = q.data {
+    if let Some(ref version) = q.data {
         let text = format!("You chose: {version}");
 
         // Tell telegram that we've seen this query, to remove 🕑 icons from the
         // clients. You could also use `answer_callback_query`'s optional
         // parameters to tweak what happens on the client side.
-        bot.answer_callback_query(q.id).await?;
+        bot.answer_callback_query(&q.id).await?;
 
         // Edit text of the message to which the buttons were attached
-        if let Some(message) = q.message {
-            bot.edit_message_text(message.chat().id, message.id(), text).await?;
+        if let Some(message) = q.regular_message() {
```

Comment:
Maybe there should be something about `MaybeInaccessibleMessage`? Or will it just clutter the code with unnecessary information

Reply:
What do you mean?

Reply:
There is a chance it will become `InaccessibleMessage`, should it be mentioned, because I changed `.message` to `.regular_message()`, or should it be simple? I like examples to share a lot of info, but maybe it's too much

Reply:
Why did you change `q.message` to `q.regular_message()`? For syntax sugar to work?

Reply:
Yup, either way you can't edit an inaccessible message, but u can delete the commit

Reply:
There's almost nowhere in the examples for syntactic sugar to be added, although i can create an example only for them

Reply:
I think this example is fine for now, we can edit it later if needed.